dwave/system/samplers/clique.py

```diff
@@ -159,6 +160,23 @@ class DWaveCliqueSampler(dimod.Sampler):
         **config:
             Keyword arguments, as accepted by :class:`.DWaveSampler`
 
+    .. versionadded:: 1.29.0
+        Support for context manager protocol.
+
+    Note:
+        The preferred and recommended way to use :class:`DWaveCliqueSampler` is
+        from a runtime context:
```

Comment:
```suggestion
        The recommended way to use :class:`DWaveCliqueSampler` is from a 
        `runtime context <https://peps.python.org/pep-0343/>`_:
```

if you want to get fancy. By the way PEP 343 says: 'for the concrete object created by the context expression and “runtime context” or (**preferably**) “runtime environment'

Reply:
If you think a link to python docs is needed here, I'd prefer https://docs.python.org/3/reference/datamodel.html#with-statement-context-managers. It's lot more succinct, and links to that PEP for more details.